instrumentation/external-annotations/README.md

```diff
@@ -0,0 +1,6 @@
+# Settings for the external annotations instrumentation
+
+| System property 	| Type 	| Default 	| Description 	|
+|-----------------	|------	|---------	|-------------	|
+| `otel.instrumentation.external-annotations.include` | String	| Default annotations | Configuration for trace annotations, in the form of a pattern that matches `'package.Annotation$Name;*'`.
```

Comment:
Although technically correct, this is useless in its current form :) We should file an issue to follow up with proper explanation for this.

Reply:
Shall we keep it like this for the time being?

Reply:
Yeah, why not.

Reply:
@theletterf please create an issue for this